src/main/java/org/folio/bulkops/processor/HoldingsDataProcessor.java

```diff
@@ -122,14 +134,16 @@ private void validateReplacement(UpdateOptionType option, Action action) throws
       }
 
       if (Set.of(PERMANENT_LOCATION, TEMPORARY_LOCATION).contains(option)) {
-        try {
-          itemReferenceService.getLocationById(newId);
+        var tenant = getTenantFromAction(action);
```

Comment:
Looks this is duplication. var tenant = getTenantFromAction(action); can be calculated outside if (Set.of(PERMANENT_LOCATION, TEMPORARY_LOCATION).contains(option)) statement.

Reply:
Fixed.